airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -144,40 +146,39 @@ def get_job_state(self, job_name: str, run_id: str) -> str:
         job_run = glue_client.get_job_run(JobName=job_name, RunId=run_id, PredecessorsIncluded=True)
         return job_run['JobRun']['JobRunState']
 
+
     def print_job_logs(
         self,
         job_name: str,
         run_id: str,
+        log_client,
```

Comment:
This seems not backward compatible to me

Reply:
Do you mean passing log_client as a parameter will create a compatibility issue?

Reply:
I mean, anyone who is currently calling this method (without `log_client` then) is going to fail once this code released

Reply:
agree, I have made change and opening new Pull request since this branch get corrupt due to multiple changes and not able to merge with all changes.

Reply:
https://github.com/apache/airflow/pull/26269

Reply:
@vincbeck please review new pull request changes at https://github.com/apache/airflow/pull/26269